src/Sarif.FunctionalTests/v2/ConverterTestData/PREfast/Expected-101.xml.sarif

```diff
@@ -12,6 +12,21 @@
           "mimeType": "text/x-cpp"
         }
       },
+      "logicalLocations": {
+        "bad1": {
+          "decoratedName": "?bad1@@YAXPAHIPAU_FLEXARRAY@@@Z"
+        },
+        "bad2": {
+          "decoratedName": "?bad2@@YAPAU_FLEXARRAY@@I@Z"
+        },
+        "Foo::what": {
+          "name": "what",
```

Comment:
Yes, this is the one that originally caught my eye. Funny how I missed it so many times in the preceding files.